swap_meet/clothing.py

```diff
@@ -1,2 +1,8 @@
-class Clothing:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+
+class Clothing(Item):
+    def __init__(self, category="Clothing", condition=0):
+        super().__init__(category, condition)
```

Comment:
This `super` call would be the ideal place to pass `category="Clothing"`, but you'd need to swap the arguments to use `category` as a named parameter.